components/dashboard/src/components/podkit/buttons/Button.tsx

```diff
@@ -28,6 +28,7 @@ export const buttonVariants = cva(
                 sm: "h-8 rounded-md px-3 text-xs",
                 lg: "h-10 rounded-md px-8",
                 icon: "h-9 w-9 p-0 rounded-sm",
+                "full-width": "h-9 px-4 py-2 w-full",
```

Comment:
I'm happy to talk about this size. We probably need some spin on `width: 100%`, because a lot of our UI depends on it. We used to name it `block`, but I found that a bit confusing and not easily parseable.